.eslintrc.json

```diff
@@ -20,13 +20,16 @@
     "curly": [2, "multi-line"],
     "global-require": 0,
     "implicit-arrow-linebreak": 0,
+    "import/prefer-default-export": 0,
     "import/no-dynamic-require": 0,
     "import/no-unresolved": [2, { "commonjs": true }],
     "new-cap": 0,
     "no-console": 0,
+    "no-named-as-default": 0,
     "no-param-reassign": 0,
     "no-shadow": [2, { "allow": ["req", "res", "err"] }],
     "no-undef": 0,
+    "no-underscore-dangle": 0,
```

Comment:
Updating rules to suit your style is not efficient. These rules are meant to reserve code quality. Please undo this and follow the lining rules